File: 217-contains-duplicate/contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    const newArr = new Set();
    for(ele of nums){
        if(newArr.has(ele))
        return true
        newArr.add(ele)
    }
    return false ;
};

File: 217-contains-duplicate/contains-duplicate.go
func containsDuplicate(nums []int) bool {
    uniques := map[int]int{}
    for _, v:=range nums {
        _, exist := uniques[v] 
        if exist{
            uniques[v]+=1
        }else {
            uniques[v]=1
        }
    }

    for _, v := range uniques {
        if v > 1 {
            return true
        }
    }
    return false
}